docs/README.md

```diff
@@ -45,9 +45,14 @@ sao user/repo --clone
 sao gitlab:user/repo
 sao bitbucket:user/repo
 
+# Use custom git host
+sao gitlab:gitlab.mydomain.com:user/repo
+sao gitlab.mydomain.com:user/repo
```

Comment:
this doesn't work I guess 😄

Reply:
I tried it with my custom gitlab and ssh key :)
`sao gitlab.weezyu.com:weezyu/api --clone`
`sao gitlab:gitlab.weezyu.com:weezyu/api`

You can find these informations here: https://www.npmjs.com/package/download-git-repo